haskell-src/Concordium/Types/Accounts.hs

```diff
@@ -732,7 +732,7 @@ data AccountInfo = AccountInfo
       aiAccountCooldowns :: ![Cooldown],
       -- | The balance of the account that is available for transactions.
       aiAccountAvailableAmount :: !Amount,
-      -- | Flag indicating whether the account is currently suspended.
+      -- | Flag indicating whether the account is currently suspended. A suspended account is in effect not participating in the consensus protocol. The aiAccountIsSuspended flag does not have any effect on stake or delegators of a validator. The full design proposal can be found on confluence under [Suspend Inactive Validators].
```

Comment:
As a style point, try to avoid lines longer than 100 characters. Also, I don't think it's good to reference confluence (which is not public) from the codebase (which is).

Reply:
Thanks for flagging this, I overlooked the soft-wrap my editor was doing there. I removed the confluence reference, it would be nice to point to design documents though somehow.